src/app/vault/groupings.component.html

```diff
@@ -73,7 +73,7 @@ <h2>{{'collections' | i18n}}</h2>
                     <ng-template #recursiveCollections let-collections>
                         <li *ngFor="let c of collections" [ngClass]="{active: c.node.id === selectedCollectionId}">
                             <a href="#" appStopClick appBlurClick (click)="selectCollection(c.node)">
-                                <i class="fa-li fa" title="{{'toggleCollapse' | i18n}}" aria-hidden="true"
+                                <i *ngIf="c.children.length" class="fa-li fa" title="{{'toggleCollapse' | i18n}}" aria-hidden="true"
```

Comment:
I do not have, nor know how to create child collections, so this portion is untested with that case.